src/bokeh/io/notebook.py

```diff
@@ -35,6 +37,7 @@
 from uuid import uuid4
 
 ## External imports
+from typing_extensions import TypeAlias
 if TYPE_CHECKING:
     from ipykernel.comm import Comm
```

Comment:
```suggestion
if TYPE_CHECKING:
    from ipykernel.comm import Comm
    from typing_extensions import TypeAlias
```

Reply:
yup this is the correct solution, had a brain freeze earlier I guess!